client/pages/sections/protocols/section.js

```diff
@@ -37,8 +37,8 @@ class Section extends PureComponent {
               <ReviewFields
                 fields={fields}
                 values={values}
-                prefix={this.props.prefix}
-                editLink={`0#${this.props.prefix}`}
+                prefix={prefix}
+                editLink={`0#${prefix}`}
```

Comment:
Not strictly necessary for the fix, but seemed weird to use the full props path when we've already used the destructured `prefix` above.